web/js/decks.v2.js

```diff
@@ -97,6 +97,10 @@ function decks_upload_all() {
   $('#archiveModal').modal('show');
 }
 
+function get_card_list_item_html(card, quantity) {
+  return '<li>'+quantity+'x '+card.title+' (<span class="small icon icon-' + card.pack.cycle.code + '"></span> ' + card.position + ')</li>';
```

Comment:
```suggestion
  return '<li>' + quantity + 'x ' + card.title + ' (<span class="small icon icon-' + card.pack.cycle.code + '"></span> ' + card.position + ')</li>';
```
Extra spacing helps readability.